osgp/protocol-adapter-iec61850/osgp-protocol-adapter-iec61850/src/main/java/org/opensmartgridplatform/adapter/protocol/iec61850/infra/networking/services/commands/Iec61850EnableReportingCommand.java

```diff
@@ -66,7 +65,7 @@ public void enableReportingOnDevice(final Iec61850Client iec61850Client, final D
      *             In case writing of data-attributes fails.
      *
      */
-    public void enableReportingOnDeviceWithoutUsingSequenceNumber(final Iec61850Client iec61850Client,
+    public void enableBufferedReportingOnDeviceWithoutUsingSequenceNumber(final Iec61850Client iec61850Client,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "iec61850Client". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1172)